pkg/openmetadata-connector-core/connector_helper.go

```diff
@@ -89,7 +91,11 @@ func (s *OpenMetadataAPIService) addTags(ctx context.Context, c *client.APIClien
 
 func (s *OpenMetadataAPIService) PrepareOpenMetadataForFybrik() bool { //nolint
 	ctx := context.Background()
-	c := s.getOpenMetadataClient()
+	c, err := s.getOpenMetadataClient(ctx)
```

Comment:
We should allow running multiple OpenMetadata connectors and restart them.

Reply:
I don't think that this code prevents multiple OpenMetadata connectors. Let's discuss.